contracts/token/TokenRules.sol

```diff
@@ -37,11 +35,6 @@ import "../organization/Organized.sol";
  */
 contract TokenRules is Organized {
 
-     /* Usings */
-
-    using SafeMath for uint256;
```

Comment:
because we don't use any uint operations? seems fair

Reply:
Yes.